dotnetv3/IAM/IAM_Basics_Scenario/IAM_Basics_ScenarioTests/IAM_BasicsTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace IAM_Basics_Scenario.Tests
+{
+    using Xunit;
+    using System.Threading.Tasks;
+    using Amazon.IdentityManagement;
+    using Amazon;
+    using Amazon.IdentityManagement.Model;
+    using IAM_Basics_Scenario;
+    using IAM_Basics_ScenarioTests;
+
+    [TestCaseOrderer("OrechstrationService.Project.Orderers.PriorityOrderer", "OrechstrationService.Project")]
```

Comment:
OrechstrationService -> OrchestrationService ?

Reply:
Thanks! Fixed. Also made the client object a private field so it doesn't have to be passed around.